dist/index.css

```diff
@@ -0,0 +1,102 @@
+/*!*****************************************************************!*\
+  !*** css ./node_modules/css-loader/dist/cjs.js!./src/index.css ***!
+  \*****************************************************************/
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body h1 {
+  padding: 3rem;
+  text-align: center;
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+}
+
+#todo-container {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  border: 1px #e7dfe7f9 solid;
+  width: 60%;
+}
+
+input {
+  background-color: rgb(255, 255, 255);
+  height: 2rem;
+  margin: 0.1rem;
+}
+
+input,
+textarea {
+  border: none;
+  resize: none;
+}
+
+li > input {
+  height: 1.5rem;
+}
+
+.input-btn-ccw {
+  width: 1.5rem;
+  height: 1.5rem;
```

Comment:
I can see you are using the width and height in your css multiple places with the same 1.5 rem, for this to comply with DRY I would kindly suggest to create a separare class if possible.

Reply:
Thanks for your comments.
It's done.